### PR TITLE
Week1

### PR DESCRIPTION
**Pull request by shanhaider23**

### CODE REVIEW

File: week1/excercise-1/ex1.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿// See https://aka.ms/new-console-template for more information
+string input = "world";
+string reversed = string.Empty;
+for (int i = input.Length-1; i >=0; i--)
```
Comment (Vertygo) [inline week1/excercise-1/ex1.cs:4]: nice, just what I wanted to see :)

File: week1/excercise-4/ex4.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        else
+        {
+            return Fibonacci(n - 1) + Fibonacci(n - 2);
+        }
+                   
+       
+    }
+  
+    
+        int n = 6;
```
Comment (Vertygo) [inline week1/excercise-4/ex4.cs:19]: Formating is a bit off, you can format the whole file by pressing ALT+SHIFT+F.

**Review by Vertygo — COMMENTED** [review]
Looks good ⭐

File: week1/excercise-4/ex4.cs (hunk 2 above)
Comment (shanhaider23) [reply week1/excercise-4/ex4.cs:19]: Thanks I will update it.

**Pull request merged**